core-api/src/main/java/com/optimizely/ab/odp/ODPEventManager.java

```diff
@@ -63,6 +66,16 @@ public ODPEventManager(@Nonnull ODPApiManager apiManager, @Nullable Integer batc
         this.flushInterval = (flushInterval != null && flushInterval > 0) ? flushInterval : DEFAULT_FLUSH_INTERVAL;
     }
 
+    // these user-provided common data are included in all ODP events in addition to the SDK-generated common data.
+    public void setUserCommonData(@Nullable Map<String, Object> commonData) {
```

Comment:
It looks confusing, at line 42 and 43 there is a NonNull tag and here it is nullable. I think it should be NonNull to keep it consistent. Same goes for line 75.